### ISSUE TITLE
Make demo mode accessible to screenreader users

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#9441]

**Comment by TKDickson** [comment]
I can log into demo mode for [iOS](https://drive.google.com/file/d/1qyVj7W14OQe_U8lSWXLGOrCHhLqVS7S_/view?usp=drive_link) and [Android](https://drive.google.com/file/d/1jq00wJ83oKqsZYv03v8T_UWAmI1Ip2XB/view?usp=sharing) with screen reader.

@brea11y this is ready for you to look, the build is "9441 demo mode accessible"

**Comment by TKDickson** [comment]
I forgot that @brea11y is out this afternoon & tomorrow - marking as blocked. Will ping about this, when she's back on Tuesday. Still have plenty of time to make it into this sprint.

**Comment by brea11y** [comment]
@TKDickson – I can also access demo mode with Talkback and VoiceOver. We should be good to go!

**Comment by TKDickson** [comment]
@artsymartha68 and @rtwell since I know you're both monitoring this ticket - it's merged into develop now, so will be included in the 2.37 release that's out to veterans at the end of September (9/24).

**Comment by rtwell** [comment]
woohoo! tyvm!
